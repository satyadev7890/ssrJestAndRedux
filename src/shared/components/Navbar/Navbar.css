.navContainer {
    width: 100%;
    align-items: center;
    display: flex;
    background-color: #666666;
    padding: 10px;
}

.navTitle {
    width: 50%;
    padding: 10px;
    padding-left: 30px;
    color: #ffffff;
    font-size: 18px;
    font-family: "fLucida Console";
}

.navItem {
    padding: 10px;
    color: #ffffff;
    font-size: 17px;
}

body {
    display: block;
    margin: 0px;
    font-family: Arial, Helvetica, sans-serif;
}