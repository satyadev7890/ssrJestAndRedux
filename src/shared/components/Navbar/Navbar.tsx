import React from 'react'
import { NavLink } from 'react-router-dom'
import './Navbar.css';

export default function Navbar () {
  const languages = [{
    name: 'All',
    param: 'all'
  }, {
    name: 'JavaScript',
    param: 'javascript',
  }, {
    name: 'Ruby',
    param: 'ruby',
  }, {
    name: 'Python',
    param: 'python',
  }, {
    name: 'Java',
    param: 'java',
  }]

  return (
    <div className="navContainer">
      <div className="navTitle">Software languages</div>
      {languages.map(({ name, param }) => (
        <div className="navItem" key={param}>
          <NavLink id={name} style={{textDecoration: "none", color: '#ffffff'}} activeStyle={{fontWeight: 'bold', color: '#e29e9e'}} to={`/Home/${param}`}>
            {name}
          </NavLink>
        </div>
      ))}
    </div>
  )
}