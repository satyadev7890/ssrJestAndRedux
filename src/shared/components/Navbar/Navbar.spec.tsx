import * as React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Navbar from './Navbar';

configure({ adapter: new Adapter() });

describe('Navbar snapshot testing', () => {
    it('create snapshot with theme', () => {
        const navbar = shallow(<Navbar />);
        expect(navbar).toMatchSnapshot();
    });
});
