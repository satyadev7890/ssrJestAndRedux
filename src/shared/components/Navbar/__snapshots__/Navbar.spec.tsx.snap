// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot testing create snapshot with theme 1`] = `ShallowWrapper {}`;
